doc/getting-started/Kotlin-Versions.md

```diff
@@ -0,0 +1,15 @@
+## Kotlin versions and the Gradle Kotlin DSL 
+
+Gradle ships with Kotlin DSL support which uses embedded Kotlin compiler, plus fixed Kotlin `stdlib` and `reflect` dependencies. For example, Gradle 4.3 ships with Kotlin 1.1.51, as well as `stdlib` and `reflect` with the same version. The `kotlin` package from those modules is visible through the Gradle classpath.
+
+For both backward and forward compatibility, the [compatibility guarantees](https://kotlinlang.org/docs/reference/compatibility.html) provided by Kotlin apply.
+
+### Backward compatibility
+
+Our approach is to only do backwards-breaking Kotlin upgrades on a major Gradle release. We will always clearly document which Kotlin version we ship and announce upgrade plans before a major release. 
```

Comment:
Is this guaranteed currently as the Kotlin DSL is currently in the `pre-release` state?

Reply:
Good point, this needs to be clarified too. I'll come back to this.

Reply:
Done, see e70f9c3